common/prettify.pm

```diff
@@ -370,9 +370,10 @@ sub new ($)
     my $basename = shift;
     my $buildname = shift;
     my $rev_link = shift;
+    my $scoreboard_title = shift;
 
     my $path = substr($basename, 0, index($basename, '/'));
-    my $filename = $path . "/Failed_Tests.html";
+    my $filename = $path . "/" . $scoreboard_title . "_Failed_Tests.html";
```

Comment:
This is assuming `$scoreboard_title` is already a valid file name. Ours is "DDS Build Scoreboard", which has spaces. Probably should replace the spaces with underscores.

Reply:
Isn't there already a variable somewhere that's the basename of the xml/html file?

Reply:
I'll check

Reply:
Looks like that's the one I used

Reply:
`$scoreboard_title`

Reply:
Actually the input xml file name (including path, if any) is stored in the local variable `$inp_file`. It is not used after the file is loaded. I could store it in the global variable, and parse it to extract the file name. Would this be a better approach, than using `$scoreboard_title`?

Reply:
Clarification: `$scoreboard_title` is based on  -t command line option, "the title for the scoreboard (default Scoreboard) ".

Reply:
My preference would be for the file names to be based on the `-f` argument to be consistent with other generated files.

Reply:
Is it part of the XML structure or is it the XML file name itself?

Reply:
<scoreboard>
  <group>
    <name>Group 1</name>
    <build>
      <name>dds_doc_Latest_Micro_bee_win7_vc12_i1d0o1s1m1</name>
    </build>
    <build>
      <name>dds_doc_ace6tao2_hornet_win7-x64_vc14_i1d1o0w1</name>
    </build>
  </group>
</scoreboard>

Reply:
Actually `-o` may be more accurate, but they are typically related.  The `-o` option determines output, so if the user asks for `-o path/to/name.html` then this could create file `path/to/name_extended_new_view.html` (with my made-up names replaced by relevant names).